ui/src/app/settings/components/repo-details/repo-details.tsx

```diff
@@ -37,6 +37,14 @@ export const RepoDetails = (props: {repo: models.Repository; save?: (params: New
             }
         ];
 
+        if (repository.type === 'git') {
+            items.push({
+                title: 'Bearer token (optional, for BitBucket Data Center only)',
```

Comment:
Please change to bitbucket server

Reply:
Bit Bucket Server is no longer supported, Bit Bucket Data Center is the new equivalent:
https://www.gitkraken.com/blog/bitbucket-server-transition-tips